adcircpy/forcing/tides/tpxo.py

```diff
@@ -1,91 +1,99 @@
 import os
-import pathlib
+from os import PathLike
+from pathlib import Path
 import sys
 
 from netCDF4 import Dataset
 import numpy as np
 from scipy.interpolate import griddata
 
+from adcircpy.forcing.tides.dataset import TidalDataset
 
-class TPXO:
 
-    def __init__(self):
+class TPXO(TidalDataset):
+    CONSTITUENTS = ['M2', 'S2', 'N2', 'K2', 'K1', 'O1', 'P1', 'Q1', 'Mm', 'Mf',
+                    'M4', 'MN4', 'MS4', '2N2', 'S1']
+    DEFAULT_PATH = Path(sys.executable).parent.parent / 'lib' / 'h_tpxo9.v1.nc'
```

Comment:
Please change this to use [appdirs](https://pypi.org/project/appdirs/).user_data_dir('tpxo') to keep it consistent across packages.